roles/mastodon-nginx/tasks/main.yml

```diff
@@ -24,8 +20,14 @@
     src: /etc/nginx/sites-available/{{ ansible_nodename }}.conf
     dest: /etc/nginx/sites-enabled/{{ ansible_nodename }}.conf
     state: link
-- name: Start nginx
+- name: Stop nginx for now
+  service: name=nginx state=stopped
+- name: Generate standalone Let's Encrypt TLS certificate for Mastodon instance
+  shell: certbot certonly -n --agree-tos --standalone -d {{ LOCAL_DOMAIN }} --email "webmaster@{{ LOCAL_DOMAIN }}"
```

Comment:
Sorry, I see this one currently depends upon my local domain PR #3. Can rework to not have that if #3 isn't wanted but this is.

Reply:
I just merged #3 so this one is fine as well!